<!-- launch in virtualenv 'mask_rcnn' -->

<launch>

  ####################################
  ## mask_generator.launch
  #### Auther: Miaoding (m.dai AT u.northwestern.edu DOT com)
  #### Dependencies: 
  ####   1. mask_rcnn (https://github.com/matterport/Mask_RCNN)
  #### Function:
  ####   Run and respawn 'mask_generator' node.
  #### Notes:
  ####   1. launch this in virtualenv 'mask_rcnn';
  ####   2. Currently no decent solutions for dying 'mask_generator' node, so just keep respawning it;
  ####################################
  
  <!-- mask generator -->
  <node name="mask_generator" pkg="unknown_pick" type="mask_generator.py" respawn="true" output="screen" />

</launch>